benchmark/run_distance_method.sh

```diff
@@ -10,25 +10,17 @@
 
 # usage: run the Distance Method algorithm
 set -eu
+source $(dirname "$0")/utils.sh
 args=(
-  working_dir
-  query_species_coding_seqs_fp
-  target_proteomes_dir
-  threads
-  distance_method_install_dir
-  stdout
-  stderr
+    working_dir
```

Comment:
I think we should remove the distance method from the benchmark (at least for now). It implements an implicit phylogenetic approach, however explicit tools like Ranger-dtl are already very fast & sensitive.